Add inital user documentation (#228)
cyberus-technology/usbvfiod: docs/overview.md

# Documentation

This project offers a vfio-user server that allows you to pass USB devices

> (like storage drives, keyboards, etc.) from a host system to a Cloud

Reviewer: "like" and "etc" both indicate that there can be more. Storage drivers and keyboards are part of the group of "USB devices", so you should use "such as" instead of "like".

```suggestion
(such as storage drives, keyboards) from a host system to a Cloud
```

Author: Nice suggestion to use `such as` here, but I would prefer to keep the `, etc.` to indicate there can be more. Storage and keyboard are tested in CI, but I would not prematurely put the restriction in the documentation.
